capstone/src/main/java/com/google/sps/data/dexparser/DexLoader.java

```diff
@@ -3,23 +3,27 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.FileInputStream;
+import java.io.RandomAccessFile;
 
-import java.util.Scanner;
+import java.util.zip.DataFormatException;
+import java.util.ArrayList;
 import java.util.zip.ZipEntry;
+import java.util.Scanner;
 import java.util.zip.ZipInputStream;
 import java.util.logging.Level; 
 import java.util.logging.Logger; 
-import java.util.logging.*; 
 
-/*Initiates parsing of the DEX files
+
+/* Initiates parsing of the DEX files
  *
- *This class can run on its own or its methods can be called 
+ * This class can run on its own or its methods can be called 
  */
 public class DexLoader {
 
   private static final Logger LOGGER = Logger.getLogger(DexLoader.class.getName());
+  private static ArrayList<RandomAccessFile> rafs;
```

Comment:
I would recommend having this variable declared only within the methods instead of one static. It would be more thread safe. Otherwise the risk is if two threads / requests attempt to check two separate dex files, the output would become corrupt on both requests.

Reply:
Okay. I just wanted it to be easily accessible by the whole class but that is a good suggestion to implement
Thanks!